docs/pages/versions/unversioned/sdk/sqlite.mdx

```diff
@@ -358,6 +359,58 @@ export default function App() {
 }
 ```
 
+### Sharing a database between apps/extensions (iOS)
+
+To share a database with other apps/extensions in the same App Group you can use shared containers. Follow these steps:
+
+<Step label="1">
+Configure the App Group in `app.json`:
+```json app.json
+{
+  "expo": {
+    "ios": {
+      "bundleIdentifier": "com.myapp",
+      "entitlements": {
+        "com.apple.security.application-groups": [
+          "group.com.myapp"
+        ]
+      }
+    }
+  }
+}
+```
+</Step>
+<Step label="2">
+Use [`Paths.appleSharedContainers`](filesystem-next/#applesharedcontainers) from the [`expo-file-system`](filesystem-next/) module to retrieve the path to the shared container:
```

Comment:
```suggestion

</Step>

<Step label="2">

Use [`Paths.appleSharedContainers`](filesystem-next/#applesharedcontainers) from the [`expo-file-system`](filesystem-next/) library to retrieve the path to the shared container:
```

We prefer using ["library" over module](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#glossary) as per our writing style guide